CMakeLists.txt

```diff
@@ -124,6 +124,7 @@ option(USE_POLY          "Use LibPoly for polynomial arithmetic")
 # check the intalled system version. If the user provides a directory we
 # immediately fail if the dependency was not found at the specified location.
 set(GLPK_DIR          "" CACHE STRING "Set GLPK install directory")
+set(PYTHONIC_PATH     "" CACHE STRING "Set Pythonic API repository path")
```

Comment:
Does it have to be an absolute path or does it work with relative too? If not, we have to make sure to check that it's an absolute path.